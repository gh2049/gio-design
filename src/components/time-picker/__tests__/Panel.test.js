import React from 'react';
import { mount, render } from 'enzyme';
import Panel from '../index';

describe('Testing Combobox', () => {
  //   it('should match alert base snapshot.', () => {
  //     const wrapper = render(<Panel />);
  //     expect(wrapper).toMatchSnapshot();
  //   });
  //  props test
  it('props test1', () => {
    const onClickMock = jest.fn();
    expect(
      mount(
        <Panel defaultOpenValue={false} use12Hours isAM onAmPmChange={onClickMock} onChange={onClickMock} />
      ).exists('.gio-time-picker-panel-combobox')
    ).toMatchSnapshot();
  });

  it('props test2', () => {
    const onClickMock = jest.fn();
    expect(
      mount(
        <Panel defaultOpenValue={false} use12Hours isAM onAmPmChange={onClickMock} onChange={onClickMock} disabledHours={[1,2,3]} />
      ).exists('.gio-time-picker-panel-combobox')
    ).toMatchSnapshot();
  });

  it('props test3', () => {
    const onClickMock = jest.fn();
    expect(
      mount(
        <Panel defaultOpenValue={false} use12Hours isAM={false} onAmPmChange={onClickMock} onChange={onClickMock} disabledHours={[1,2,3]} />
      ).exists('.gio-time-picker-panel-combobox')
    ).toMatchSnapshot();
  });
});
