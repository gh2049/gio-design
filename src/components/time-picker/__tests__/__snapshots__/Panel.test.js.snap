// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Combobox props test1 1`] = `false`;

exports[`Testing Combobox props test2 1`] = `false`;

exports[`Testing Combobox props test3 1`] = `false`;
